### PR TITLE
HPCC-22230 Lang Ref COMBINE examples should have LOCAL

### PR DESCRIPTION
**Pull request by JamesDeFabia**
Signed-off-by: Jim DeFabia <jamesdefabia@lexisnexis.com>
@ghalliday Please review (Link to PDF below)

## Type of change:
- [ ] This change is a bug fix (non-breaking change which fixes an issue).
- [ ] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change improves the code (refactor or other change that does not change the functionality)
- [ ] This change fixes warnings (the fix does not alter the functionality or the generated code)
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x] My code follows the code style of this project.
  - [ ] My code does not create any new warnings from compiler, build system, or lint.
- [x] The commit message is properly formatted and free of typos.
  - [ ] The commit message title makes sense in a changelog, by itself.
  - [ ] The commit is signed.
- [ ] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [x] I have read the CONTRIBUTORS document.
- [x] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [ ] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.
- [ ] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [ ] This is a user interface / front-end modification
  - [ ] I have tested my changes in multiple modern browsers
  - [ ] The component(s) render as expected

## Smoketest:
- [ ] Send notifications about my Pull Request position in Smoketest queue.
- [ ] Test my draft Pull Request.

## Testing:

Rendered PDF:
http://10.240.32.243/view/Docs%207.0/job/JimDocBuild7/CE=ubuntu-18.04-amd64-1/78/artifact/build-docs/Release/docs/EN_US/ECLLanguageReference_EN_US-7.2.17-closedown0.pdf#page=166

### CODE REVIEW

**Comment by ghalliday** [comment]
@JamesDeFabia 
why are there so many unrelated changes.  It looks like all the lines have been reformatted to be < 80 characters long - which seems a bit archaic .  It makes it hard to spot the items that have actually changed.

**Comment by JamesDeFabia** [comment]
The original examples were indented badly so I reformatted them with Richard Taylor's help. They were not made to be <80 characters, just made to be more easily readable. 
(we do have a 94 character max for our PDF output, but that is just FYI)
The only salient code change was adding LOCAL to the COMBINE statements.

File: docs/EN_US/ECLLanguageReference/ECLR_mods/BltInFunc-COMBINE.xml [hunk 1]
```diff
@@ -209,57 +241,63 @@ nameRec := RECORD
 END;
  
 resultRec := RECORD(inrec)
-  DATASET(nameRec) names;
+  DATASET(nameRec)
```
Comment (ghalliday) [inline docs/EN_US/ECLLanguageReference/ECLR_mods/BltInFunc-COMBINE.xml]: Why the newline here?

**Comment by ghalliday** [comment]
@JamesDeFabia one comment on the indentation changes.  Otherwise looks good.

File: docs/EN_US/ECLLanguageReference/ECLR_mods/BltInFunc-COMBINE.xml (hunk 1 above)
Comment (JamesDeFabia) [reply docs/EN_US/ECLLanguageReference/ECLR_mods/BltInFunc-COMBINE.xml]: @gavin I removed the newline. Back to you

**Comment by JamesDeFabia** [comment]
@ghalliday I made the change you requested, this should be good now

**Pull request merged**